test/integration/crud/find_and_modify.test.js

```diff
@@ -318,4 +320,57 @@ describe('Find and Modify', function () {
     expect(error.message).to.match(/must not contain atomic operators/);
     await client.close();
   });
+
+  context('when passed an ObjectId instance as the filter', () => {
+    let client;
+    let findAndModifyStarted;
+
+    beforeEach(async function () {
```

Comment:
```suggestion
    beforeEach(function () {
```
This doesn't need to be async